woocommerce/woocommerce-ios
# Login: New Prologue Screens

Fixes: https://github.com/woocommerce/woocommerce-ios/issues/3824

## Description

Updates the current Prologue screen with new illustrations in a carousel to promote the following features:

- Stats
- Order Management
- Products
- Reviews

## Changes

The new Prologue screen and carousel has a few parts:

* `LoginPrologueViewController.swift` is still the main VC, with an updated background image and colors, and unused methods removed. `LoginPrologueViewController.xib` was also updated with these changes. (The RBG values for the color **Gray 0** are also updated to the correct values for this shade.)
* `LoginProloguePageViewController.swift` is the UIPageViewController, with a custom page control, embedded as a child view of `LoginPrologueViewController`. (This is modeled after [the UIPageViewController in WordPressAuthenticator](https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/blob/develop/WordPressAuthenticator/Signin/LoginProloguePageViewController.swift).)
* `LoginProloguePages.swift` contains the carousel's unique content (images and text) in `LoginProloguePageType` and shared styles/layout in `LoginProloguePageTypeViewController`.

### Screenshots

/|Stats|Order Management|Products|Reviews
-|-|-|-|-
Light|![Simulator Screen Shot - iPhone 11 Pro - 2021-03-26 at 13 32 39](https://user-images.githubusercontent.com/8658164/112639998-d3f08480-8e38-11eb-888d-560dd9477c59.png)|![Simulator Screen Shot - iPhone 11 Pro - 2021-03-26 at 13 32 41](https://user-images.githubusercontent.com/8658164/112640009-d652de80-8e38-11eb-8280-641c3178ad47.png)|![Simulator Screen Shot - iPhone 11 Pro - 2021-03-26 at 13 32 43](https://user-images.githubusercontent.com/8658164/112640012-d6eb7500-8e38-11eb-8efb-45a7f6490565.png)|![Simulator Screen Shot - iPhone 11 Pro - 2021-03-26 at 13 32 46](https://user-images.githubusercontent.com/8658164/112640015-d6eb7500-8e38-11eb-8341-7e5f3125d621.png)
Dark|![Simulator Screen Shot - iPhone 11 Pro - 2021-03-26 at 13 33 06](https://user-images.githubusercontent.com/8658164/112640022-d9e66580-8e38-11eb-863d-01eb6799c7d0.png)|![Simulator Screen Shot - iPhone 11 Pro - 2021-03-26 at 13 33 08](https://user-images.githubusercontent.com/8658164/112640023-da7efc00-8e38-11eb-83a3-216adc75bdae.png)|![Simulator Screen Shot - iPhone 11 Pro - 2021-03-26 at 13 33 10](https://user-images.githubusercontent.com/8658164/112640027-db179280-8e38-11eb-9bd2-b78ab6c919ce.png)|![Simulator Screen Shot - iPhone 11 Pro - 2021-03-26 at 13 33 12](https://user-images.githubusercontent.com/8658164/112640032-db179280-8e38-11eb-8675-e347e266763c.png)


iPhone SE|iPhone 11|iPad Pro (11")
-|-|-
![Simulator Screen Shot - iPhone SE (2nd generation) - 2021-03-26 at 13 32 18](https://user-images.githubusercontent.com/8658164/112640047-df43b000-8e38-11eb-9724-a8dc37c44e32.png)|![Simulator Screen Shot - iPhone 11 Pro - 2021-03-26 at 13 32 39](https://user-images.githubusercontent.com/8658164/112640054-e10d7380-8e38-11eb-9290-d1ad0b00ab6c.png)|![Simulator Screen Shot - iPad Pro (11-inch) (2nd generation) - 2021-03-26 at 13 31 30](https://user-images.githubusercontent.com/8658164/112640066-e4a0fa80-8e38-11eb-8329-ad382dae4dc0.png)



## Testing

1. Start logged out of the app.
2. Build and run the app, and confirm the initial (prologue) screen shows a carousel containing four pages with the new illustrations.
3. Confirm the layout displays all of the images/text, on both small and large device sizes.
4. Confirm the carousel and illustrations are reversed if the app is using a right-to-left language (e.g. Arabic, Hebrew).

## Submitter Checklist

Update release notes:
